crawler.rb

```diff
@@ -2,10 +2,16 @@
 
 require 'fileutils'
 
+mode = ARGV.shift || ""
+
+unless mode.empty?
+  mode = "-#{mode}"
+end
+
 # FileUtils.rm_rf('data')
 
 system("sudo apt-get install mdbtools")
 
-system("relaton fetch-data status-smg-3GPP")
+system("relaton fetch-data status-smg-3GPP#{mode}")
```

Comment:
I'm not sure if a direct call Ruby method will work here, because the `mbtools` is installed after the Ruby script is loaded, but let's try to use `Relaton3gpp::DataFetcher.fetch("status-smg-3GPP#{mode}")` instead `system` call.
To be able call the Ruby method we need to add `require "relaton_3gpp"` after the `system("sudo apt-get install mdbtools")` line.

Reply:
Done